src/librustdoc/html/static/js/search.js

```diff
@@ -1723,12 +1717,17 @@ window.initSearch = function(rawSearchIndex) {
             `${typeFilter}</h1> in ${crates} </div>`;
         if (results.query.error !== null) {
             output += `<h3>Query parser error: "${results.query.error}".</h3>`;
+        } else if (results.query.foundElems <= 1 && results.query.returned.length === 0) {
+            output += `<div id="titles">` +
+                makeTabHeader(0, "In Names", ret_others[1]) +
+                makeTabHeader(1, "In Parameters", ret_in_args[1]) +
+                makeTabHeader(2, "In Return Types", ret_returned[1]) +
+                "</div>";
+        } else {
+            output += '<div id="titles">' +
+                makeTabHeader(0, "In Function Signature", ret_others[1]) +
```

Comment:
Should we differentiate between `a -> b` and `-> b`?

Reply:
"In function signature" and "In function returned value"? Also, does "function" include "method" as well?

Reply:
I guess you're right.